src/bin/inx-chronicle/process.rs

```diff
@@ -1,28 +1,19 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-pub async fn interrupt_or_terminate() {
+pub async fn interrupt_or_terminate() -> eyre::Result<()> {
     #[cfg(unix)]
     {
         use tokio::signal::unix::{signal, SignalKind};
-
-        let mut sigterm = signal(SignalKind::terminate()).expect("cannot listen to `SIGTERM`");
-        let mut sigint = signal(SignalKind::interrupt()).expect("cannot listen to `SIGINT`");
-
+        let mut terminate = signal(SignalKind::terminate())?;
+        let mut interrupt = signal(SignalKind::interrupt())?;
         tokio::select! {
-            _ = sigterm.recv() => {
-                tracing::info!("received `SIGTERM`, sending shutdown signal")
-            }
-            _ = sigint.recv() => {
-                tracing::info!("received `SIGINT`, sending shutdown signal")
-            }
+            _ = terminate.recv() => {}
```

Comment:
Please don't remove these logs.

Reply:
we already log this in the caller. It's also wrong, because how does this function know what's going on outside of this function.